### PR TITLE
Improve env cleanup

### PR DESCRIPTION
**Pull request by sharbov**
- Use docker-compose down command to cleanup
- Remove kill_continaers & remove_containers methods
- Remove skipper network hack
- Expose ports for containers which need external access

### CODE REVIEW

**Comment by sharbov** [comment]
**Background:**

We wanted to use the build container as the test container, but it was not a part of the `docker-compose.yaml` file and it was managed by skipper, so they didn't share the same network.
We "fixed" that by adding the `skipper.yaml` network configuration (a.k.a "the hack").

That change caused the `docker-compose down` command to fail, because the build container was still connected to the test network while we ran it. so instead of using it we killed and removed the containers manually. 

That manual clean-up left garbage behind, like docker volumes and networks.
This change was introduced to solve that.

Please note that this is a *major* change and it's not backward compatible.

File: README.md [hunk 1]
```diff
@@ -101,67 +101,68 @@ class ExampleTest(BaseDockerTest):
...
-        self.wiremock = WiremockController(url='http://mocked.service:9999')
+        self.wiremock = WiremockController(url='http://localhost:9999')
         self.addCleanup(self.wiremock.reset_mapping)
 
     def test_services_sanity(self):
         """Validate services are responsive once the test start."""
-        logging.info('Validating consul container is responsive')
-        self.assertEquals(requests.get('http://consul.service:8500').status_code, httplib.OK)
+        log.info('Validating consul container is responsive')
+        self.assertEquals(requests.get('http://localhost:8500').status_code, http_client.OK)
```
Comment (eran-stratoscale) [inline README.md]: Now that it's just localhost:port, please use const for the port number .

**Comment by ravidbro** [comment]
+1

**Comment by ravidbro** [comment]
update service template to use that

**Pull request merged**